Fixed problem with sketch running slow

dchao99 opened the pull request:
Removed the inefficient digitalPinToBitMask() and digitalPinToPort(), which was causing the sketch to run very slow if compiled by later versions of Arduino. 
Disable interrupts inside SetWhiteBal(), so the timer ISR can’t come in at the same time, it will screw up the control signal timing.

lincomatic commented:
thanks for the PR. Could you please flatten it to just one commit? It's too hard to go through all the changes

dchao99 commented:
I think I have squashed all the commits into one.
Other things I added:
- Add a function to change the interrupt timer counter
- Add a define REVERSE_ROWCHANNELS to flip the column order into the shift registers.
